src/pages/index.js

```diff
@@ -78,34 +111,103 @@ picturePopup.setEventListeners();
 const userInfo = new UserInfo({
   nameSelector: "#profile-title",
   jobSelector: "#profile-description",
+  avatarSelector: "#profile-image",
+});
+
+api.getCurrentUser().then((result) => {
+  userInfo.setUserInfo(result.name, result.about);
+  userInfo.setAvatar(result.avatar);
 });
 
 /******************
  * EVENT HANDLERS *
  ******************/
 function handleProfileEditSubmit(e, { title, description }) {
   e.preventDefault();
-  userInfo.setUserInfo(title, description);
-  profilePopup.close();
+  profilePopup.loading(true);
+  api
+    .updateUser(title, description)
+    .then((res) => res.json())
```

Comment:
`.then((res) => res.json())`


Please, delete it

* You need to check responses only in `Api` like this: ` .then(this._checkResponse)`. It's responsibility of this class

**Please, correct it everywhere**